src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -270,8 +272,17 @@ async def pex_from_targets(
                     "file does not cover all requirements."
                 )
             else:
-                requirements = PexRequirements(str(req) for req in constraints_file_reqs)
-                description = description or f"Resolving {python_setup.requirement_constraints}"
+                repository_pex = await Get(
+                    Pex,
+                    PexRequest(
+                        description=f"Resolving {python_setup.requirement_constraints}",
```

Comment:
Should probably keep the `description or` part.

Reply:
I don't think so, but check me on this: This description is always correct right here. The user-provided description now just flows down to the final - actually intended - PexRequest below that uses this full resolve.